README.md

```diff
@@ -1,3 +1,7 @@
+:warning: **The scripts in this repository are no longer needed as of [User Activity Monitoring plugin version 1.4](https://docs.cloudbees.com/docs/release-notes/latest/plugins/user-activity-monitoring-plugin/1.4).**
```

Comment:
TODO: Verify that this link works correctly once the release is out before merging the PR.